src/VolleyManagement.Backend/VolleyManagement.Domain/TeamsAggregate/Team.cs

```diff
@@ -1,38 +1,162 @@
-﻿namespace VolleyManagement.Domain.TeamsAggregate
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using static VolleyManagement.Domain.TeamsAggregate.TeamValidation;
+using static VolleyManagement.Domain.Properties.Resources;
+
+namespace VolleyManagement.Domain.TeamsAggregate
 {
     /// <summary>
     /// Team domain class.
     /// </summary>
     public class Team
```

Comment:
I can add two same players into the team. Fix it

Reply:
@sdiachen  Validation has check for this case.

Reply:
Decrecated.